dask/dataframe/tests/test_optimize_dataframe.py

```diff
@@ -15,10 +15,8 @@
 
 
 def test_column_optimizations_with_bcolz_and_rewrite():
-    try:
-        import bcolz
-    except ImportError:
-        return
+    pytest.importorskip('bcolz')
```

Comment:
I did not have bcolz and castra installed on the virtenv i was working on, and thought having Travis run the tests instead was easier.

Sorry for the bad etiquette, I have now run the tests and they successfully passed on my machine.
The two failures on Travis are with `test_read_bytes_blocksize_on_large_data` and `test_from_s3` seem completely unrelated and fail on other PRs.